_studio/mfx_lib/encode_hw/vp9/src/mfx_vp9_encode_hw_vaapi.cpp

```diff
@@ -548,10 +544,12 @@ mfxStatus SetFrameRate(
 
     frameRateBufIds.resize(numTL);
 
-    mfxU32 base_framerate;
-    PackMfxFrameRate(par.mfx.FrameInfo.FrameRateExtN, par.mfx.FrameInfo.FrameRateExtD, base_framerate);
+    mfxU32 num = par.mfx.FrameInfo.FrameRateExtN;
```

Comment:
Note. N stands for nOminator, so "nom" would be proper name for the variable.

Reply:
Doesn't N stand for numerator?

Reply:
Nominator - denominator.